### ISSUE TITLE
.diggsml file extension

### ISSUE THREAD

**Issue by nickmachairas** [DIGGSml/schema-dev#32]
Has it ever been considered officially adopting a file extension other than .xml for DIGGS files? Such as .diggsml, .dgsml, or similar. It would be a small but significant improvement for programs that identify, create and parse DIGGS files.

**Comment by panars** [comment]
I do not remember it being discussed before. I searched and there are some other xml schemas that use a different file extension, such as MusicXML (.mxl), .mathml, .gml, and .kml . The benefit would be automatically opening a DIGGSML file with an associated program.

**Comment by sdeaton75** [comment]
I agree - it definitely would have the benefit being able to be opened automatically/recognized as a DIGGS file without having to open it and is fundamentally a good suggestion.

**Comment by dponti** [comment]
I like this idea but am not personally a fan of file extensions that are longer than 4 characters, although we may not have much choice if we want to create a unique but recognizable extension as ones I would normally think of, such as .dgml, .dml, .dxml .dsml and .diml are already in use (although it is not uncommon for different file types to use the same extension).

Perhaps this thread can generate some discussion on appropriate extension names? I'm ok with either of Nick's suggestions but perhaps .dgsml would be better as it's shorter.

**Comment by panars** [comment]
While 3 or 4 characters are pretty typical for file extensions, there are plenty of extensions with more than 4 characters. ACCDB and BLEND are two that come to mind. I think if we shorten it too much it may cause more confusion. My feeling is that it is okay to be longer than 4 characters if it helps clarify what the file is for.  Does it have to include "ml" for markup language? I tend to think not. I like just plain ".diggs" because that is how we have been talking about these files anyway, e.g.  "Can you convert that to a DiGGS file?"

**Comment by sdeaton75** [comment]
I agree with Peter - I would just go with .diggs.
